model.py

```diff
@@ -6147,6 +6150,12 @@ def get(cls, _db, url, do_get=None, extra_request_headers=None,
         representation.content = content
         return representation, False
 
+    @classmethod
+    def post(cls, _db, url, params, max_age=None):
```

Comment:
First, this should be called `cachable_post` or something. Most of the time the responses to POST requests are not cachable, and I don't want someone to call this thinking it's just a shortcut for making a POST request.

Second, by currying the `simple_http_post` method here you can take unsightly POST-specific code out of `get()`. Something like this:

```
def do_post(*args, *kwargs):
    request_kwargs = {'data' : params}
    kwargs.update(request_kwargs=request_kwargs)
    cls.simple_http_post(*args, **kwargs)

return cls.get(_db, url, do_get=do_post, max_age=max_age)
```

Does this make sense?